src/Dfc.CourseDirectory.Database/Pttcd/Views/vwProviderSearchIndex.sql

```diff
@@ -0,0 +1,17 @@
+create or alter view Pttcd.vwProviderSearchIndex
+as
+select
+    p.ProviderId,
```

Comment:
Do we want a mechanism to remove a Provider from the index?

Reply:
Good question, that brings up a whole load more questions for me:

* The old index didn't have `IsDeleted` configured. I don't know if we have some other mechanism for removing things from the index currently.
* I don't know what the desired behaviour is around any kind of previously active provider; given the provider search shows a status column do we want to include unregistered providers? I also don't know what the ukprn status means and how it can change. I'll have a look and see if I can figure out what it does now when a provider is "[Unregistered](https://github.com/SkillsFundingAgency/dfc-coursedirectory/blob/ecae97aab7fea6aae10bab155043912117c22305/src/Dfc.CourseDirectory.Core/Models/ProviderStatus.cs#L7)". I'll also have a look at the behaviour around ukprn status.
* I don't know if there are circumstances where the provider record is hard-deleted currently and whether the existing index does anything with that, and whether we should handle it here somehow.

Reply:
There is no way to delete a provider *at all* currently. I don't think we need to worry about it here yet.